models/movement.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const mongoose = require('mongoose');
+
+const movementSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  description: {
+    type: String,
+    maxlength: 200
+  },
+  startYear: String,
+  endYear: String,
+  ancestor: String
```

Comment:
Almost... 'ancestor` should be a reference from one Movement to another Movement;
e.g. 
"Impressionism is the ancestor of Pointillism"
Therefore, it should be `_ancestor` and have the type of Mongoose Schema ID.